yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -836,6 +836,9 @@ private[spark] class Client(
           sys.env.get(envname).foreach(env(envname) = _)
         }
       }
+      // pass conf spark.pyspark.driver.python to AM in cluster mode
+      sparkConf.getOption(SparkLauncher.PYSPARK_DRIVER_PYTHON)
```

Comment:
Is this needed? Won't `PythonRunner` pick this up from the conf when the AM starts?